crates/jstzd/tests/octez_node_test.rs

```diff
@@ -0,0 +1,53 @@
+#![cfg(not(target_os = "macos"))]
+// This test is disabled for macos because octez binary is not available
```

Comment:
could we default to look for binaries on `PATH`? Then at least its runnable locally for us.

Reply:
This line is removed since the test is ignored now anyway.